### PR TITLE
adding prom metrics for operations

### PR DESCRIPTION
**Pull request by rmartinez3**
### What does this PR achieve? Why do we need it?
Adding prom metrics for stale operations,in flight, failed, new and total.
Useful to expose more metrics and to setup alerts when a stale operation occurs.

### CODE REVIEW

File: pkg/metrics/metrics.go [hunk 1]
```diff
@@ -133,6 +163,11 @@ func (m *Metrics) Describe(ch chan<- *prometheus.Desc) {
 	ch <- deviceFreeInBytes
 	ch <- deviceUsedInBytes
 	ch <- brickCount
+	ch <- staleCount
```
Comment (phlogistonjohn) [inline pkg/metrics/metrics.go]: It would be nice to have a comment here delineating the "sub sections" of items.

File: pkg/metrics/metrics.go [hunk 2]
```diff
@@ -158,6 +193,37 @@ func (m *Metrics) Collect(ch chan<- prometheus.Metric) {
 		prometheus.GaugeValue,
 		float64(len(topinfo.ClusterList)),
 	)
+
+	opinfo, err := m.app.AppOperationsInfo()
+	if err != nil {
+		log.Println("Can't collect Opertions info for metrics: " + err.Error())
```
Comment (phlogistonjohn) [inline pkg/metrics/metrics.go]: "Opertions" -> "Operations"

File: pkg/metrics/metrics.go (hunk 1 above)
Comment (rmartinez3) [reply pkg/metrics/metrics.go]: Added comment

File: pkg/metrics/metrics.go (hunk 2 above)
Comment (rmartinez3) [reply pkg/metrics/metrics.go]: Fixed typo

**Comment by rmartinez3** [comment]
Fixed Tests. Let me know if anything else is needed.

**Comment by phlogistonjohn** [comment]
@rmartinez3 changes look good thanks! Could you please squash all your changes into a single commit and then I think I will be able to approve the PR. Thank you!

**Pull request merged**